google/cloud/aiplatform/training_jobs.py

```diff
@@ -1150,6 +1150,31 @@ def __init__(
                 "set using aiplatform.init(staging_bucket='gs://my-bucket'"
             )
 
+    @classmethod
+    def get(
+        cls,
+        training_job_name: str,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
```

Comment:
I couldn't add a type annotation as discussed in https://github.com/python/typing/issues/254

I could add it after adding `from __future__ import annotations` but this broke something else.

Reply:
Let me know what you want to use this workaround (from the above link):

```
_T = TypeVar("_T")

class Foo:
    @classmethod
    def create(cls: Type[_T]) -> _T:
        return cls()
```

Reply:
Ah I just went with ` -> "CustomTrainingJob"`